lib/processor/steps/crop.js

```diff
@@ -1,20 +1,35 @@
 var helpers = require('../../helpers');
+var _ = require('lodash');
 
 module.exports = function(context, stepInfo) {
   var img = context.processedImage;
   helpers.dimension.resolveStep(img, stepInfo);
-
-  var anchorX = stepInfo.anchor && stepInfo.anchor.length === 2 && stepInfo.anchor[1];
-  var anchorY = stepInfo.anchor && stepInfo.anchor.length === 2 && stepInfo.anchor[0];
+  var anchor = stepInfo.anchor || 'cc';
+  var anchorX = anchor.length === 2 ? anchor[1] : 'c';
+  var anchorY = anchor.length === 2 ? anchor[0] : 'c';
 
   if (isNaN(stepInfo.top)) {
     stepInfo.top = 0;
+
+    if (stepInfo.anchorY) {

```

Comment:
All this logic could probably be moved to `resolveStep`: https://github.com/asilvas/node-image-steam/blob/master/lib/helpers/dimension.js

That way other methods could use anchoring in the future. If nothing else make use of the `getInfo` helper.